### PR TITLE
Replace bash-style DR scripts with openshift-tests run-dr

### PR DESCRIPTION
**Pull request by vrutkovs**
Requires https://github.com/openshift/origin/pull/23208

This commit would replace bash-style DR CI scenarios with a proper e2e test. Several helper scripts for bastion_ssh were retained for e2e tests which use ssh bastion (see #4161)

### CODE REVIEW

**Comment by vrutkovs** [comment]
/hold

until Origin PR merges

**Comment by vrutkovs** [comment]
These tests were added in https://github.com/openshift/origin/pull/23208, but they are not listed in `openshift-tests run openshift/disruptive --dry-run`.

This is fixed in https://github.com/openshift/origin/pull/23520

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -306,316 +306,18 @@ objects:
...
-          export PATH=$PATH:/tmp/google-cloud-sdk/bin
-          mkdir gcloudconfig
-          export CLOUDSDK_CONFIG=/tmp/gcloudconfig
-          gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
-          gcloud config set project openshift-gce-devel-ci
-        }
-
         function run-dr-snapshot-tests() {
-          openshift-tests run-dr restore-snapshot "${TEST_SUITE}" \
+          openshift-tests run all --dry-run | grep dr-etcd-snapshot | openshift-tests run all -f - \
```
Comment (deads2k) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: not beautiful. but we could do worse.

**Comment by deads2k** [comment]
looks like an improvement with less bash. That grep one liner rubs me the wrong way though.

/approve

**Comment by petr-muller** [comment]
Stale PR, please adapt and reopen if still needed.

/close

**Pull request closed without merging**